ambry-network/src/main/java/com/github/ambry/network/http2/Http2NetworkClient.java

```diff
@@ -124,6 +133,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
     }
     http2ClientMetrics.http2ClientSendTime.update(System.currentTimeMillis() - startTime);
     // TODO: close stream channel for requestsToDrop. Need a hashmap from corelationId to streamChannel
+    if (requestsToDrop.size() != 0) {
+      logger.warn("Number of requestsToDrop: {}", requestsToDrop.size());
+      http2ClientMetrics.http2RequestsToDropCount.inc(requestsToDrop.size());
+    }
 
     http2ClientResponseHandler.getResponseInfoQueue().poll(readyResponseInfos, pollTimeoutMs);
```

Comment:
nit: you might be able to use releaseAndCloseStreamChannel in warmUpConnections too.